src/api/Procfile

```diff
@@ -1,5 +1,5 @@
-web: bundle exec rails server -b 0.0.0.0
-delayed: bundle exec script/delayed_job.api.rb run
-clock: bundle exec clockworkd --log-dir=log -l -c config/clock.rb run 
-search: bundle exec rake ts:rebuild NODETACH=true
+web: (bundle check || bundle install) && bundle exec rails server -b 0.0.0.0
+delayed: (bundle check || bundle install) && bundle exec script/delayed_job.api.rb run
+clock: (bundle check || bundle install) && bundle exec clockworkd --log-dir=log -l -c config/clock.rb run
+search: (bundle check || bundle install) && bundle exec rake ts:rebuild NODETACH=true
```

Comment:
Is it not a problem that all these services get started at the same time? So potentially you execute bundle install 4 times? Can we not just do it in the docker-compose file:

```
command: (bundle check || bundle install) && foreman start -p 3000
```

Reply:
I'm not sure if it will cause issues, but moving it from this file to the Docker Compose file doesn't change much. The command is still executed.

Reply:
Once, not 4 times :-)